evadb/plan_nodes/abstract_plan.py

```diff
@@ -126,5 +126,5 @@ def find_all(self, plan_type: Any):
         """
 
         for node in self.bfs():
-            if isinstance(node, plan_type):
+            if isinstance(node, plan_type) or self.opr_type == plan_type:
```

Comment:
What case we are trying to take care here?

Reply:
No special case. I just feel the `plan_type` here can be the actual instance type or the internal `opr_type`, so I just added it to cover both cases. I can also remove it.

Reply:
Could you give an example `isinstance(node, plan_type)` does not work? I am trying to understand what do you mean  by internal `opr_typle`?